src/lib/Dungeon.js

```diff
@@ -353,6 +353,56 @@ class AutomationDungeon
                 return;
             }
 
+            // If flash is unlocked, use it to avoid fighting every encounters
+            if (DungeonRunner.map.flash)
+            {
+                const floor = DungeonRunner.map.playerPosition.peek().floor;
+                const currentBoard = DungeonRunner.map.board.peek()[floor];
+                const unwrapedTiles = currentBoard
+                    .flatMap((row, i) => row.map((tile, j) => ({ tile, i, j })))
+                    .reduce((acc, { tile, i, j }) => ({...acc, [`${i},${j}`]: {tile, i, j}}), {});
+                const visibleUnvisitedTiles = Object.values(unwrapedTiles).filter(({ tile }) => tile._isVisible && !tile._isVisited);
```

Comment:
avoid using pokeclicker internal variable, prefer using the public interface:
```suggestion
                const visibleUnvisitedTiles = Object.values(unwrapedTiles).filter(({ tile }) => tile.isVisible && !tile.isVisited);
```

Reply:
I was sure that the `isVisible` did not exist, that is why I was using the internal one. But it is indeed there